### PR TITLE
Works and about me section

### PR DESCRIPTION
**Pull request by deyemiobaa**
- Build two sections for works and about me
- Style both sections
- Add background images and icons

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -117,15 +119,268 @@ a {
...
+  margin-bottom: 62px;
 }
 
 .socials li {
   padding: 10px 10px 10px 10px;
 }
 
+.recent-works {
+  display: flex;
+  flex-direction: column;
```
Comment (RolandM99) [inline style.css]: - [ ] The main container of this section should have a `display: grid;`  kindly, change the flex by the CSS grid. Let's rock, fix it 🧑‍🔧

File: index.html [hunk 2]
```diff
@@ -47,10 +47,154 @@ <h1>Hey there. I'm Sodiq. <br> <span>I'm a Software Developer</span></h1>
...
+    </section>
+
+    <section class="recent-works"> <!-- flexbox container-->
+      <div class="arrow"> <!-- flexbox item-->
         <i class="fa-solid fa-circle-chevron-down"></i>
       </div>
+
+      <div class="title"> <!-- flexbox item-->
+        <p>My Recent Works</p>
+      </div>
```
Comment (RolandM99) [inline index.html]: - [ ] The title 'My Recent Works' should uses a `hx` tag. please, Kindly fix it to the header tag `h1` .

File: index.html [hunk 3]
```diff
@@ -47,10 +47,154 @@ <h1>Hey there. I'm Sodiq. <br> <span>I'm a Software Developer</span></h1>
...
+
+      <div class="bar"></div> <!-- flexbox item-->
+
+      <div class="recent-works-cards">  <!-- flexbox item-->
+        <div class="works work1"> <!-- flexbox container-->
+          <div class="work-details"> <!--Grid container-->
+            <p>Multi-Post Stories Gain+Glory</p>
```
Comment (RolandM99) [inline index.html]: - [ ] Referencing the project requirements, each project title should use a header tag` hx` (h1, h2, h3,..) . Please, change the` p` tag with a` hx` tag of your choice.

File: style.css [hunk 4]
```diff
@@ -117,15 +119,268 @@ a {
...
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  padding: 8px 8px;
+  background: #ebf0ee;
+  border-radius: 4px;
+}
+
+.works-button {
+  padding: 12px;
+  width: 98px;
```
Comment (RolandM99) [inline style.css:242]: - [ ] Please, Add the button styling (Hover, enable, Active, Pressed ) 
Figma design :
image

File: index.html [hunk 5]
```diff
@@ -47,10 +47,154 @@ <h1>Hey there. I'm Sodiq. <br> <span>I'm a Software Developer</span></h1>
...
+            <a href="#" class="works-button">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="about-me">
+
+      <div class="about-me-text">
+        <p class="about-me-title">About me</p>
```
Comment (RolandM99) [inline index.html]: - [ ] The title 'About me' should uses a header tag `hx` (h1, h2,...). Kindly, fix it! 
"_you're almost there, the little details make the beauty of things_"

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @deyemiobaa  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there, your project looks very good!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 
- Descriptive PR ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @deyemiobaa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**